paien.yaml

```diff
@@ -612,6 +612,18 @@ packages:
         ^py-numpy+blas+lapack@1.14.3
         ^magma@2.3.0+fortran+shared
         ^cuda@9.1.85
+        ^python@2.7.14+tk
+
+  gnu-gpu-lapack-python-libraries:
+    target_matrix:
+      - gnu-gpu
+    requires:
+      - architecture
+      - compiler
+      - python
+      - lapack
+    specs:
+      - py-numpy+blas+lapack@1.14.3
```

Comment:
Do we want this or:
```
py-numpy+blas+lapack@1.14.3 ^magma@2.3.0+fortran+shared ^cuda@9.1.85
```
on GPU nodes?

Reply:
It seems to me that py-numpy does not depend on magma. It's py-pytorch (a few lines above) which depends on magma (which depends on cuda), which is just above py-pytorch in the list.

Reply:
@rmsds Double checked, I think you're right. I thought `magma` was a `lapack` provider for GPUs, but it is not apparently. Then this is fine with me. I'll approve feel free to merge. Otoh `py-numpy` should have been activated along with `py-torch`.